groups/sig-release/groups.yaml

```diff
@@ -400,7 +400,7 @@ groups:
       - monmonmsc@gmail.com # 1.20 Bug Triage Shadow
       - pal.nabarun95@gmail.com  # 1.20 RT Lead Shadow
       - prashsh1@in.ibm.com # 1.20 CI Signal Shadow
-      - rey.lejano@rx-m.com # 1.20 Docs Shadow
+      - rlejano@gmail.com # 1.21 Docs Lead
```

Comment:
@reylejano -- Can you please remove your email ID from here? This alias is specifically for shadows.

Reply:
Removed email ID and squashed commits